### PR TITLE
standardize_SyncLocalDir_comments

### LINKED ISSUE #364: [DOC]363
**Provide a link to that doc page:**
#363 
**What is the defect and your suggestions on improvement:**

### PR DESCRIPTION
**Pull request by zyt123456789**
### Ⅰ. Describe what this PR does
standardize the comments(pkg/ddc/alluxio/operations/local.go).

### Ⅱ. Does this pull request fix one issue?

fixes #364 

None
### Ⅲ. List the added test cases (unit test/integration test) if any, please explain if no tests are needed.
No

### Ⅳ. Describe how to verify it
See the describe

### Ⅴ. Special notes for reviews

### CODE REVIEW

File: pkg/ddc/alluxio/operations/local.go [hunk 1]
```diff
@@ -20,11 +20,9 @@ import (
 	"time"
 )
 
-/*
-* Sync local path by running command `du -sh <path>`.
-* Under the circumstance where some NAS(e.g. NFS) is mounted on the `<path>`, the function will sync metadata of all files in the NAS.
-* This is necessary for Alluxio to get consistent file metadata with UFS(i.e. NAS in this case).
- */
+//SyncLocalDir Sync local path by running command `du -sh <path>`.
```
Comment (cheyang) [inline pkg/ddc/alluxio/operations/local.go:23]: Sync local -> syncs local

**Review by cheyang — CHANGES_REQUESTED** [review]

File: pkg/ddc/alluxio/operations/local.go (hunk 1 above)
Comment (zyt123456789) [reply pkg/ddc/alluxio/operations/local.go:23]: Sync local -> sync local  您是不是多打了一个s

Comment (zyt123456789) [reply pkg/ddc/alluxio/operations/local.go]: 已经修改，谢谢呀

**Comment by zyt123456789** [comment]
您好！已经修改完毕。



------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "fluid-cloudnative/fluid"                                                                                    <notifications@github.com&gt;;
发送时间:&nbsp;2020年11月10日(星期二) 下午5:25
收件人:&nbsp;"fluid-cloudnative/fluid"<fluid@noreply.github.com&gt;;
抄送:&nbsp;"731849160"<731849160@qq.com&gt;;"Author"<author@noreply.github.com&gt;;
主题:&nbsp;Re: [fluid-cloudnative/fluid] standardize_SyncLocalDir_comments (#363)





 
@cheyang requested changes on this pull request.
 
 
In pkg/ddc/alluxio/operations/local.go:
 &gt; @@ -20,11 +20,9 @@ import (  	"time"  )   -/* -* Sync local path by running command `du -sh <path&gt;`. -* Under the circumstance where some NAS(e.g. NFS) is mounted on the `<path&gt;`, the function will sync metadata of all files in the NAS. -* This is necessary for Alluxio to get consistent file metadata with UFS(i.e. NAS in this case). - */ +//SyncLocalDir Sync local path by running command `du -sh <path&gt;`.  
Sync local -&gt; syncs local

**Review by cheyang — APPROVED** [review]

**Pull request merged**